plugins/modules/nios_nsgroup_forwardingmember.py

```diff
@@ -0,0 +1,173 @@
+    description:
+      - Allows for the configuration of Extensible Attributes on the
+        instance of the object.  This argument accepts a set of key / value
+        pairs for configuration.
+    required: false
+    type: str
+  comment:
+    description:
+      - Configures a text string comment to be associated with the instance
+        of this object.  The provided text string will be configured on the
+        object instance.
+    required: false
+    type: str
+  state:
+    description:
+      - Configures the intended state of the instance of the object on
+        the NIOS server.  When this value is set to C(present), the object
+        is configured on the device and when this value is set to C(absent)
+        the value is removed (if necessary) from the device.
+    choices: [present, absent]
+    default: present
+    type: str
+'''
+
+EXAMPLES = '''
+- name: create infoblox nameserver forwarding member group
+  infoblox.nios_modules.nios_nsgroup_forwardingmember:
+    name: my-forwardingmember-group
+    comment: "this is a forwarding member nameserver group"
+    forwarding_servers:
+      - name: member.localdomain
+    state: present
+    provider:
+      host: "{{ inventory_hostname_short }}"
+      username: admin
+      password: admin
+  connection: local
+
+- name: create infoblox nameserver forwarding member group with external forwarders
+  infoblox.nios_modules.nios_nsgroup_forwardingmember:
+    name: my-forwardingmember-group
+    comment: "this is a forwarding member nameserver group with external forwarders"
+    forwarding_servers:
+      - name: member.localdomain
+        use_override_forwarders: true
+        forward_to:
+          - name: ns1
+            address: 192.168.10.1
+    state: present
+    provider:
+      host: "{{ inventory_hostname_short }}"
+      username: admin
+      password: admin
+  connection: local
+'''
+
+RETURN = ''' # '''
+
+from ansible.module_utils.basic import AnsibleModule
+from ..module_utils.api import WapiModule
+from ..module_utils.api import NIOS_NSGROUP_FORWARDINGMEMBER
+
+
+def main():
+    '''entrypoint for module execution.'''
+
+    argument_spec = dict(
+        provider=dict(required=True),
+        state=dict(default='present', choices=['present', 'absent']),
+    )
+
+    forward_to_spec = dict(
+        name=dict(required=True),
+        address=dict(required=True),
+    )
+
+    forwarding_servers_spec = dict(
+        name=dict(required=True),
+        forwarders_only=dict(type='bool', default=False),
+        forward_to=dict(type='list', elements='dict', options=forward_to_spec, default=[], required=False),
+        use_override_forwarders=dict(type='bool', default=False),
+    )
+
+    ib_spec = dict(
+        name=dict(required=True, ib_req=True),
+        forwarding_servers=dict(type='list', elements='dict', required=True, options=forwarding_servers_spec),
+        extattrs=dict(),
+        comment=dict(),
+    )
+
+    argument_spec.update(ib_spec)
```

Comment:
Please change `argument_spec.update(ib_spec)` to `argument_spec.update(normalize_ib_spec(ib_spec))`. This fixes the sanity error.